pallets/gear/src/lib.rs

```diff
@@ -373,7 +371,7 @@ pub mod pallet {
                                     });
 
                                     // TODO: weight to fee calculator might not be identity fee
```

Comment:
I guess the comment is not actual anymore since gas converting is defined in the Config now?

Reply:
Fair enough